src/index.css

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
```

Comment:
Please make sure to embed this font files from Google Fonts or from file declared on a `@font-face {}` statement